packages/sdk/src/utils/customChains.ts

```diff
@@ -60,7 +60,41 @@ export const ancient8Testnet = {
   },
 } as const satisfies Chain
 
+export const scroll = {
```

Comment:
It looks like wagmi has scroll. Do we need a custom chain?

Reply:
Yep added a ticket to upgrade wagmi/viem later. We're a bit behind.